ee/tabby-webserver/src/service/background_job/third_party_integration/issues.rs

```diff
@@ -92,5 +92,22 @@ pub async fn index_gitlab_issues(
         index.add(doc).await;
     }
 
+    let merge_requests: Vec<GitlabIssue> = gitlab::api::paged(
+        MergeRequests::builder().project(full_name).build()?,
+        gitlab::api::Pagination::All,
+    )
+    .query_async(&gitlab)
+    .await?;
+
+    for merge_request in merge_requests {
+        let doc = WebDocument {
+            id: format!("gitlab/{full_name}/merge_requests/{}", merge_request.id),
```

Comment:
will it be easier to just use web url as I'd?

Reply:
Probably, shall I do the same for other issues?

Reply:
yes